### ISSUE TITLE
aws-ssm: CloudFormation failing to update custom resource that references ssm SendCommandCommand

### ISSUE THREAD

**Issue by tdq132** [aws/aws-cdk#27806]
### Describe the bug

When adding in a custom resource that calls SSM `SendCommandCommand`, somewhere, this is being translated to `SendCommandCommandCommand` (i.e. an extra `Command` is being added to the SDK call. 

In my `cdk.out`, I can see the custom resource with `sendCommandCommand`, as intended.

On CloudFormation deploy, the custom resource fails to deploy with error: `Received response status [FAILED] from custom resource. Message returned: Unable to find command named: SendCommandCommandCommand for action: SendCommandCommand in service package @aws-sdk/client-ssm`

### Expected Behavior

I expect the custom resource to call SSM `SendCommandCommand`, which is the correct SDK call. 

### Current Behavior

Custom resource calling SSM `SendCommandCommand` is being rewritten to `SendCommandCommandCommand` which is nonexiestent in the AWS SDK JS v3 library. 

### Reproduction Steps

``` python 
custom_resources.AwsSdkCall(
    action="sendCommandCommand",
    service="SSM",
    parameters=send_command_parameters,
    physical_resource_id=custom_resources.PhysicalResourceId.of(
        f"{document}{instance}"
    ),
    output_paths=["Command.CommandId"],
)
```

### CDK CLI Version

2.103.1 (build 3bb19ac)

### Node.js Version

v20.6.1

### OS

Ubuntu 23.04

### Language

Python

### Language Version

Python 3.11.4

**Comment by pahud** [comment]
According to the [doc](https://github.com/aws/aws-cdk/tree/68630c728848429b7430e8ba4d9f21c50b2d1b52/packages/aws-cdk-lib/custom-resources#using-aws-sdk-for-javascript-v3), have you tried this in the latest AWS CDK release?

```
service: 'ssm',
action: 'sendCommand',
```

**Comment by tdq132** [comment]
Yes, `sendCommand` works, but this does not explain why it has changed from `sendCommand`, to `sendCommandCommand`, and now back to `sendCommand`.
